console/backend/src/main/java/nl/nn/adapterframework/management/web/RequestMessageBuilder.java

```diff
@@ -85,7 +100,18 @@ public static RequestMessageBuilder create(FrankApiBase base, BusTopic topic, Bu
 	}
 
 	public Message<?> build() {
-		SEC_LOG.always().log(createLogMessage());
+		if(SEC_LOG.isInfoEnabled()) {
+			String method = base.getServletRequest().getMethod();
+			String issuedBy = sanitizeForLog(HttpUtils.getCommandIssuedBy(base.getServletRequest()));
+			if((method.equalsIgnoreCase("GET") || method.equalsIgnoreCase("OPTIONS"))) {
+				SEC_LOG.debug("created bus request from URI [{}:{}] issued by{}", method, base.getUriInfo().getRequestUri(), issuedBy);
```

Comment:
A few points on the above code:
1. Line 103: `.isInfoEnabled` -> and you log on debug level at line 107 but on 112 on info level. Should it be `.isDebugEnabled` or logging should be on info-level at line 107?
2. The log text on line 107 and 112 are missing a white space at `by{}`

Reply:
1. If loglevel == DEBUG, it also logs at INFO level. This check covers both levels.
2. the issuedBy string starts with a space.

Reply:
Ok